Visma.net/Models/AttributeDetails.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ONIT.VismaNetApi.Lib;
+using ONIT.VismaNetApi.Models.CustomDto;
+using ONIT.VismaNetApi.Models.Enums;
+
+namespace ONIT.VismaNetApi.Models
+{
+    public class AttributeDetails : DtoProviderBase
```

Comment:
Should this have an override for PrepareForUpdate? Or is it not possible to update attributes?

Reply:
Missed will fix that.

Reply:
Looked at it now and I actually did it in the Attribute class. (models\Attribute.cs)

Reply:
Good!